main.js

```diff
@@ -0,0 +1,11 @@
+var saveButton = document.querySelector("button");
```

Comment:
@ben-rosner-williamsburg Do you think we should have a class or id on that button, since we're going to have other buttons in the future? Will this bite us later when we try to do some sort of event handling on another button?

Reply:
@ericahagle Yes I agree that we should have something there.